extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1733,11 +1750,11 @@ static public function getGroups() {
 	 */
 	static public function getVariables( $sort = "cv_name", $wiki = 0, $group = 0,
 		$defined = false, $editable = false, $string = false ) {
-		if ( ! self::isUsed() ) {
+		if ( ! static::isUsed() ) {
 			Wikia::log( __METHOD__, "", "WikiFactory is not used." );
 			return false;
 		}
-		$dbr = self::db( DB_MASTER );
+		$dbr = static::db( DB_MASTER );
 
 		$aVariables = [];
 		$tables = [ "city_variables_pool", "city_variables_groups" ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "city_variables_pool" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)